prediction-applier/src/main/java/org/alfresco/hxi_connector/prediction_applier/repository/NodesClient.java

```diff
@@ -110,6 +111,7 @@ public void configure()
             .when(header(HTTP_RESPONSE_CODE).isNotEqualTo(String.valueOf(EXPECTED_STATUS_CODE)))
                 .process(this::throwExceptionOnUnexpectedStatusCode)
             .otherwise()
+                .log(DEBUG, log, "Node updated successfully - Headers: ${headers}, Body: ${body}")
```

Comment:
I'm not sure if we should log message headers/body at `debug` level - they may contain sensitive information (I am not sure if we should log that at all, but if so - maybe at `trace`)
Besides that - should we delete/change log message 3 lines lower?

Reply:
This was temporary solution to investigate logs - it's already fixed.
I can revert the minimal d-c file 🤔

Reply:
Minimal d-c file has been restored, but in a minimalist version - with references to main d-c setup